dags/vm_resource.py

```diff
@@ -173,7 +171,7 @@ class ClusterName(enum.Enum):
   V5E_16_MULTISLICE_CLUSTER = "v5e-16-bodaborg"
   V5E_256_MULTISLICE_CLUSTER = "v5e-256-bodaborg"
   V5E_256_US_WEST_4_MULTISLICE_CLUSTER = "v5e-256-bodaborg-us-west4"
-  BODABORG_V6E_256_EUROPE_WEST4_A = "bodaborg-v6e-256-europe-west4-a"
+  BODABORG_V6E_256 = "bodaborg-v6e-256"
   A3_CLUSTER = "maxtext-a3-20n"
```

Comment:
Similar change in https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/393, cc @raymondzouu

Reply:
@raymondzouu I am making the changes in this PR as my tests weren't working on the previously used cluster.